packages/additionalapi/src/database/ImportLogic.ts

```diff
@@ -250,13 +250,14 @@ export async function storeNodes(client: PoolClient, repositoryData: RepositoryD
     }
 }
 
-async function storeNodesThroughFlatBuffers(client: PoolClient, repositoryData: RepositoryData, dbConnection: DbConnection, bulkImport: FBBulkImport, repositoryName: string)
+async function storeNodesThroughFlatBuffers(client: PoolClient, repositoryData: RepositoryData, dbConnection: DbConnection, bulkImport: FBBulkImport)
```

Comment:
I cannot add a suggestion as the corresponding lines were not touched, but to make this work with bulkImport we need to change 8 instances of the call to COPY (4 in storeNodes and 4 in storeNodesThroughFlatBuffers)

From:
```
await pipeInputIntoQueryStream(client, `COPY "${repositoryName}".....
```
To:
```
await pipeInputIntoQueryStream(client, `COPY "${SCHEMA_PREFIX}${repositoryName}".....
```

We also need to import `SCHEMA_PREFIX`

Reply:
Not sure what you mean, but there is no need to import SCHEMA-PREFIX, we can use the `schema_name` property of RepositoryData, instead of using the `repository_name` property.

This (storeNodesThroughFlatBuffers) :

```
const repositoryName = repositoryData.repository.schema_name
await pipeInputIntoQueryStream(client, `COPY "${repositoryName}".....`
```

Becomes:

```
const repositorySchema = repositoryData.repository.repository_schema
await pipeInputIntoQueryStream(client, `COPY "${repositorySchema}".....`

```

Reply:
The RepositoryData object contains all RepositoryInfo, e.g. the `repository_name` which is the name as used by clients,  and the `schema_name`, which is used internally in the server.

Reply:
> The RepositoryData object contains all RepositoryInfo, e.g. the repository_name which is the name as used by clients, and the schema_name, which is used internally in the server.

That would be perfect

> Not sure what you mean, 

With "I cannot add a suggestion as the corresponding lines were not touched," I meant that in a PR review I can add a special comment with the code to be changed (so that you can click on it and get the change applied) but given this PR do not touch those lines I cannot comment them and therefore I cannot add the suggestion

Reply:
Now I do understand what you wrote :-).
I change the ImportLogic to use the schema name in 8 places.

Reply:
Perfect, thank you! I will mark this comment as resolved